src/electrum.rs

```diff
@@ -344,6 +344,18 @@ impl Rpc {
         }
     }
 
+    fn features(&self) -> Result<Value> {
+        Ok(json!({
+            "genesis_hash": self.tracker.chain().get_block_hash(0),
+            "hosts": {},
+            "protocol_max": PROTOCOL_VERSION,
+            "protocol_min": PROTOCOL_VERSION,
+            "pruning": null,
+            "server_version": format!("electrs {}", ELECTRS_VERSION),
```

Comment:
The (recent) doc says:

> Should be the same as the first element of the result to the server.version() RPC call.

However here's it's separated with space but [above](https://github.com/romanz/electrs/pull/445/files#diff-ede2fa6030c1edb89d5de41ac20473c31669e46197974a46108a9bff88eebfaeR333) with slash.
I suggest writing `fn server_id() -> String` and using it in both places to keep them in sync

Reply:
Good catch - thanks!